bin/sos-collector

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/python3
```

Comment:
Just a note to the name (how one can comment on a new file _name_? :) ). Doesnt it matter we have `sosreport` without the comma? Should not this be renamed to `soscollector` to have one naming logic?

I am in doubts. Coherent naming is nice, current `sos-collector` is better for eyes.